src/main/java/bisq/desktop/main/dao/wallet/send/BsqSendView.java

```diff
@@ -159,7 +153,8 @@ public void initialize() {
                                     bsqFormatter.formatCoinWithCode(receiverAmount)))
                             .actionButtonText(Res.get("shared.yes"))
                             .onAction(() -> {
-                                walletsManager.publishAndCommitBsqTx(txWithBtcFee, new FutureCallback<Transaction>() {
+                                // TODO not updated to TxBroadcaster changes because not in sync with voting branch anyway
+                               /* walletsManager.publishAndCommitBsqTx(txWithBtcFee, new FutureCallback<Transaction>() {
```

Comment:
Should this be fixed now that voting branch is being merged?

Reply:
Yes, only technical refactorings (class rename, move) in the dao project got applied.